src/profile/profile.view.ts

```diff
@@ -79,6 +80,31 @@ const thisPane: PaneDefinition = {
 
       // Todo: only show this if there is vcard info
       heading('Contact')
+
+      const chatContainer = dom.createElement('div')
+      const exists = await getChat(subject, false)
+      if (exists) {
+        chatContainer.appendChild(longChatPane.render(exists, context, {}))
```

Comment:
Yes, for the moment I think it's fine that the chat appears in the place where you click the button. I guess when you come back to a profile where you click 'chat with me' it should maybe show 'reopen the chat you started earlier' so that the actual profile doesn't move down too much. Follow-up issue: https://github.com/solid/solid-panes/issues/246 - let's merge this now and continue that work there.